main.cpp

```diff
@@ -106,150 +106,300 @@ struct CarWash //2)
 
 
 /*
- 1)
+ 1)midi keyboard
  5 properties:
- 1)
- 2)
- 3)
- 4)
- 5)
+ 1)keys
+ 2)pitch wheel
+ 3)mod wheel
+ 4)usb lead
+ 5)body
  3 things it can do:
- 1)
- 2)
- 3)
+ 1)plays midi notes
+ 2)changes pitch
+ 3)changes macros
  */
+struct MidiKeyboard
+{
+    int keys = 88;
+    bool pitchWheel = true;
+    bool modWheel = true;
+    bool usbCompatable = true;
+    bool metalBody = false;
+
+    void playMidiNotes(int keys);
+    void changePitch(bool pitchWheel);
+    void controlMacros(bool modWheel);
 
+    
+};
 /*
- 2)
+ 2)guitar
  5 properties:
- 1)
- 2)
- 3)
- 4)
- 5)
+ 1)body
+ 2)neck
+ 3)strings
+ 4)jack
+ 5)cable
  3 things it can do:
- 1)
- 2)
- 3)
+ 1)play notes
+ 2)play chords
+ 3)play muted
  */
+struct Guitar
+{
+    
+    bool body = true;
+    bool neck = true;
+    int strings = 6;
+    bool jack = true;
+    bool cable = true;
 
+    void playNote(int strings);
+    void playChord(int strings);
+    void playMuted(bool neck, int strings);
+
+};
 /*
- 3)
+ 3)mobile phone
  5 properties:
- 1)
- 2)
- 3)
- 4)
- 5)
+ 1)screen
+ 2)buttons
+ 3)mic
+ 4)speaker
+ 5)usb input
  3 things it can do:
- 1)
- 2)
- 3)
+ 1)make calls
+ 2)recieve calls
+ 3)send text messages
  */
+struct MobilePhone
+{
+    
+    bool screen = true;
+    bool buttons = true;
+    bool mic = true;
+    bool speaker = true;
+    bool usbInput = true;
```

Comment:
- [ ] come up with something other than 5 "this has a ____" properties.
examples:  Number of CPU cores, number of apps installed, number of pixels, etc...